app/components/super-features/super-features-relations/super-features-relations.html

```diff
@@ -1,7 +1,7 @@
 <div class="super-features-relations-list-header">
   <h4>Child Pages</h4>
 
-  <div ng-if="relations.length > 0">
+  <div ng-if="article.child_types.length > 0">
```

Comment:
Relations shouldn't exist without child_types. Relations are predicated on child_types existing, this is preventing the creation of new relations.